php/php.dbgp/src/org/netbeans/modules/php/dbgp/ui/DbgpLineBreakpointCustomizerPanel.java

```diff
@@ -184,6 +198,22 @@ public Controller getController() {
         return controller;
     }
 
+    private static int findNumLines(FileObject file) {
+        DataObject dataObject;
+        try {
+            dataObject = DataObject.find (file);
+        } catch (DataObjectNotFoundException ex) {
+            return 0;
```

Comment:
Do we want to add some logging here? 🤔

Reply:
It's hard to answer this question, since I took this code unchanged from https://github.com/apache/netbeans/blob/master/java/debugger.jpda.ui/src/org/netbeans/modules/debugger/jpda/ui/breakpoints/LineBreakpointPanel.java

Reply:
Maybe, I would leave a message like this: `"Can't find DataObject for " + file.getPath()`

Reply:
Done.

Reply:
Cool, thanks.